sql/core/src/test/resources/sql-tests/results/ansi/array.sql.out

```diff
@@ -489,3 +489,43 @@ select get(array(1, 2, 3), -1)
 struct<get(array(1, 2, 3), -1):int>
 -- !query output
 NULL
+
+
+-- !query
+select array_append(array(1, 2, 3), 4)
+-- !query schema
+struct<array_append(array(1, 2, 3), 4):array<int>>
+-- !query output
+[1,2,3,4]
+
+
+-- !query
+select array_append(array('a', 'b', 'c'), 'd')
+-- !query schema
+struct<array_append(array(a, b, c), d):array<string>>
+-- !query output
+["a","b","c","d"]
+
+
+-- !query
+select array_append(array(1, 2, 3), CAST(null AS INT))
+-- !query schema
+struct<array_append(array(1, 2, 3), CAST(NULL AS INT)):array<int>>
+-- !query output
+NULL
```

Comment:
The result is correct ?

Reply:
This is more consistent with other high order functions we are having.

Reply:
If any of the input is null, we are returning null for the same.

Reply:
`[1,2,3,NULL]` now

Reply:
Yes ! Changed the output as well as sql.